corehq/apps/prototype/templates/prototype/data_cleaning/partials/forms/clean_data_form.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+{% load crispy_forms_tags %}
+
+<div id="{{ container_id }}"
+     hx-swap="outerHTML"
+     hq-hx-refresh="{{ table_selector }}">
```

Comment:
why are these on the div? I expected to see them on the same element that's making the request.